README.md

```diff
@@ -56,7 +56,11 @@ var params = { shortUrl: 'http://goo.gl/xKbRu3' };
 
 // get the long url of a shortened url
 urlshortener.url.get(params, function (err, response) {
-  console.log('Long url is', response.longUrl);
+  if (err) {
+    console.log('Encountered error', err);
```

Comment:
It seems that `console.log` already adds spaces between the first and second arguments. If I add another space at the end of the string literals, I end up with two spaces between the string and the second argument.  At least this is the behaviour with node 0.12.2